Sources/StreamChatUI/ChatMessageList/ChatMessage/Decorators/ChatMessageHeaderDecoratorView.swift

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+public struct ChatMessageHeaderDecoratorViewContent {
```

Comment:
```suggestion
public struct ChatChannelMessageHeaderDecoratorViewContent {
```
As this is used from channels, shall we name it `ChatChannelMessageHeaderDecoratorViewContent`?

Reply:
Democracy wins 😄